orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/pipeline/support/SourceResolver.groovy

```diff
@@ -70,7 +70,7 @@ class SourceResolver {
     //with useSourceCapacity prefer the largest ASG over the newest ASG
     def latestAsg = stageData.useSourceCapacity ? regionalAsgs.sort { (it.instances as Collection)?.size() ?: 0 }.last() : regionalAsgs.last()
     return new StageData.Source(
-      account: stageData.account, region: latestAsg["region"] as String, asgName: latestAsg["name"] as String
+      account: stageData.account, region: latestAsg["region"] as String, asgName: latestAsg["name"] as String, serverGroupName: latestAsg["name"] as String
```

Comment:
nit: Put each entry on a new line for readability. The array brackets access is also inconsistent with how we do map lookups everywhere else in orca (using dot notation)